### PR TITLE
Nginx conf: upstreaming customizations

### PR DESCRIPTION
**Pull request by bryanlatten**
- Gzip handling based on ENV variable (can be set in downstream Dockerfile)
- Prod-level tokens

### CODE REVIEW

File: container/root/run.d/10-nginx.sh [hunk 1]
```diff
@@ -25,3 +29,9 @@ then
   echo "[nginx] server index is ${SERVER_INDEX}"
   sed -i "s/index index.html index.htm/index ${SERVER_INDEX}/" $CONFIG_SITE
 fi
+
+if [[ $SERVER_GZIP_OPTIONS ]]
```
Comment (jimdelois) [inline container/root/run.d/10-nginx.sh]: Can you name this `$SERVER_GZIP_ENABLED` or something more binary-ish?  It sounds like there are options contained within the actual variable, but it's behaving like a flag here.

Comment (bryanlatten) [inline container/root/run.d/10-nginx.sh]: discussed in real life. gzip is technically enabled by default in nginx, this is opening the gzip options to serve/process static content this way

Comment (jimdelois) [inline container/root/run.d/10-nginx.sh]: :+1: Anything else really becomes too wordy.  Fine to move forward with this.

**Comment by jimdelois** [comment]
:white_check_mark:

**Comment by mannytoledo** [comment]
:+1:
